packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * `terra-icon` documentation corrections. 
```

Comment:
nit: Can we specify that these are import corrections. Just corrections seems unclear as to what was changed/updated.

Reply:
Fixed here: 7dbae3d2047fd184887a6a46bb9b3637318183cd